src/styles/blocks/header.scss

```diff
@@ -0,0 +1,48 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+
+  &-item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &-link {
+    display: inline-block;
+    font-size: $font-size-s;
+    line-height: $line-height-s;
+    font-weight: 500;
+    text-decoration: none;
+    text-transform: uppercase;
+    padding: 23px 0;
+    color: #060b35;
+    transition: ease 0.3s;
+  }
+
+  &-link:hover {
+    color: $turquoise-color;
+  }
+}
+
+.is-active {
+  color: $turquoise-color;
+  position: relative;
+
+  &::before {
+    content: "";
+    position: absolute;
+    height: 4px;
+    background-color: $turquoise-color;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    border-radius: 8px;
+  }
+}
```

Comment:
I created a separate scss file for nav block, but I don't know if my way is right